packages/react-form-hooks/src/hooks/errorPropagation.ts

```diff
@@ -0,0 +1,28 @@
+import {useEffect} from 'react';
+import get from 'get-value';
+
+import {isField} from '../utilities';
+import {FieldOutput, FormError} from '../types';
+
+export function useErrorPropagation(
+  fieldBag: {[key: string]: FieldOutput<any>},
+  remoteErrors: FormError[],
+) {
+  useEffect(
+    () => {
+      remoteErrors.forEach(error => {
+        if (error.fieldPath == null) {
+          return;
+        }
+
+        const got = get(fieldBag, error.fieldPath);
+
+        if (isField(got)) {
+          got.setError(error.message);
+        }
```

Comment:
Seems bad to silently throw away errors you can't associate to fields?

Reply:
We don't throw them away, they stay in `submitErrors` so they can still be displayed in the banner or whatever.